Swift/Tests/CBLTestCase.swift

```diff
@@ -92,8 +92,9 @@ class CBLTestCase: XCTestCase {
     
     override func tearDown() {
         self.defaultCollection = nil
+        self.otherDB_defaultCollection = nil
         try! db.close()
-        try! otherDB?.close()
+        try? otherDB?.close()
```

Comment:
Not sure, can it be `try!`?

Reply:
I just tested, `try!` works as well... I don't remember why I put it that way.. will update